.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+
+services:
+  - docker
+
+script:
+  - make
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" == "master" ]; then
+    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
```

Comment:
Can we move the Docker commands into a Makefile target and call that here from within this if-block? It's easier to maintain the Makefile than making changes to `.travis.yml`.